docs/pages/project/releases.md

```diff
@@ -4,9 +4,198 @@ title: Releases
+
+### [v0.3.6](https://github.com/layer5io/meshery/releases/tag/v0.3.6)
+
+**What's new ?**
+- Mesheryctl
+    + Ad-hoc connectivity tests for Grafana is now supported. Users can click the Grafana chip and have Meshery verify its ability to connect to the configured Grafana instance.
+
+### [v0.3.5](https://github.com/layer5io/meshery/releases/tag/v0.3.5)
+
+**What's new ?**
+- Mesheryctl
+    + Removal of `init` as a command exposed to users. This command's functionality is used internal to mesheryctl start. A new start `--check` command will provide preflight check functionality in init's place.
+
+### [v0.3.4](https://github.com/layer5io/meshery/releases/tag/v0.3.4)
+
+**What's new ?**
+- Mesheryctl
+    + `mesheryctl version` is now enhanced with the addition of displaying the git commit (sha) of the mesheryctl release.
+
+### [v0.3.3](https://github.com/layer5io/meshery/releases/tag/v0.3.3)
+
+**What's new ?**
+- Meshery Server
+    + Providers - a new project construct that allows users to select authentication, long-term storage, and so on provider.
+
+### [v0.3.2](https://github.com/layer5io/meshery/releases/tag/v0.3.2)
+
+**What's new ?**
+- Mesheryctl
+    + Addition of `mesheryctl version` as a new subcommand.
+
+### [v0.3.1](https://github.com/layer5io/meshery/releases/tag/v0.3.1)
+
+**What's new ?**
+- Meshery Server
+    + Support for wrk2 as an alternative load generator.
+
+### [v0.2.4](https://github.com/layer5io/meshery/releases/tag/v0.2.4)
+
+**What's new ?**
+- Mesheryctl
+    + Homebrew support available for mesheryctl.
+- Meshery Server
+    + Meshery adapter for Octarine released as stable.
+- Documentation
+    + Revised quick start guide for Mac, Linux and Windows.
+    + WSL2 support published.
+    + GKE kubeconfig generation script switched to --decode.
+
+### [v0.2.3](https://github.com/layer5io/meshery/releases/tag/v0.2.3)
+
+**What's new ?**
+- Mesheryctl
+    + Improved `status` command output on Windows.
+- Meshery Server
+    + Ability to deploy Meshery on Istio.
+    + Adapter Chips: Move adapter port number into tooltip.
+
+### [v0.2.2](https://github.com/layer5io/meshery/releases/tag/v0.2.2)
+
+**What's new ?**
+- Mesheryctl
+    + Improved `update` command, no overwriting in local configuration while `start` command.
+- Meshery UI
+    + A new modal view to organize and display performance results in a tabular format.
+
+### [v0.2.1](https://github.com/layer5io/meshery/releases/tag/v0.2.1)
+
+**What's new ?**
+- Mesheryctl
+    + Overhaul of in-cluster vs out-of-cluster Kubernetes setup.
+
+## [v0.2.0](https://github.com/layer5io/meshery/releases/tag/v0.2.1)
+
+**What's new?**
+- Meshery Adapters
+    + Adapter introduced for Network Service Mesh.
+- Meshery Server
+    + Ability to run performance tests asynchronously.
+    + Collect & persist node metrics.
+
+<!-- <table class="responsive-table hover striped">
```

Comment:
Hopefully, we'll figure out a way to get releasedrafter to backfill here. If not, we can work on manually backfilling.